src/components/atoms.jsx

```diff
@@ -1,6 +1,37 @@
+import React from 'react'
 import styled, { keyframes } from 'styled-components'
 import { Link } from 'react-router-dom'
 
+const Label = styled.div`
+  font-size: 1.5rem;
```

Comment:
`NB:` Are we still using rem?

Reply:
Nope! I believe this was just copy-pasta'd from another Label. I'll switch this up